client/finality-grandpa/src/tests.rs

```diff
@@ -456,6 +459,7 @@ fn add_forced_change(
 fn finalize_3_voters_no_observers() {
 	let _ = env_logger::try_init();
 	let mut runtime = current_thread::Runtime::new().unwrap();
+	let threads_pool = futures03::executor::ThreadPool::new().unwrap();
```

Comment:
@romanb migrated tests in the external `finality-grandpa` crate over to `LocalPool`. Would that be better here also? Does it make a difference?

Reply:
The `ThreadPool` is a global threads pool where tasks are dispatched.
The difference is that with a `LocalPool` things are more deterministic, but if the code doesn't have any bug, that shouldn't change anything.